utils/src/main/scala/za/co/absa/enceladus/utils/transformations/DeepArrayTransformations.scala

```diff
@@ -62,14 +190,105 @@ object DeepArrayTransformations {
       name
     }
 
-    // Returns true if a path consists only of 1 element meaning it is the leaf element of the input column path requested by the caller
-    def isLeafElement(path: Seq[String]): Boolean = path.lengthCompare(2) < 0 // More efficient version of path.length == 1
+    def ensureOutputColumnNameValid(columnName: String): Unit= {
+      if (columnName.isEmpty) {
+        if (isAddColumn) {
+          throw new IllegalArgumentException(s"The column '$inputColumnName' already exists.")
+
+        } else {
+          throw new IllegalArgumentException(s"Output field cannot be empty when transforming an " +
```

Comment:
I guess the string concatenation is done because of the line length limit?
I think, just a suggestion, it would be better to either put the string to its own line as whole or breaking the limit.